params/config_test.go

```diff
@@ -117,25 +117,25 @@ func TestCheckCompatible(t *testing.T) {
 		},
 		{
 			stored:        TestChainConfig,
-			new:           TestPreSubnetEVMConfig,
+			new:           TestPreSubnetEVMChainConfig,
 			headBlock:     0,
 			headTimestamp: 0,
 			wantErr: &ConfigCompatError{
 				What:         "SubnetEVM fork block timestamp",
 				StoredTime:   utils.NewUint64(0),
-				NewTime:      nil,
+				NewTime:      TestPreSubnetEVMChainConfig.NetworkUpgrades.SubnetEVMTimestamp,
```

Comment:
why was this changed?

Reply:
`TestPreSubnetEVMChainConfig` now has a future time rather than having nils (to reflect VM behaviour)